app/Repository/SpeakerRepository.php

```diff
@@ -37,4 +37,24 @@ public function getLastSpeakers(int $numberOfSpeakers): array
 
         return $queryBuilder->getQuery()->getResult();
     }
+
+    public function getTotalTalks(\DatePeriod $datePeriod): int
```

Comment:
This is SpeakerRepository and this method returns speakers count, but currently named as `getTotalTalks`.

```suggestion
    public function getTotalSpeakers(\DatePeriod $datePeriod): int
```